university-of-lincoln-harvard.csl

```diff
@@ -24,7 +24,7 @@
   </locale>
   <macro name="editor">
     <names variable="editor" delimiter=", ">
-      <name and="text"/>
+      <name/>
```

Comment:
Why this change? I think the and should stay and you should add `initialize-with="."`, no? Or what are you trying to do?

Reply:
Hi Adam,

I've kind of inherited this work so I'm picking up a lot of things here as I'm going. Pretty much all of these aren't changes I'm sure I've made so I'm not entirely sure why they're needed or what implications they have! This being a prime example, I'm not sure what initialize-with would do for us.

Reply:
`<name/>` gives you 
Adam Smith, Karl Marx, Andrew Beeken.

`<name and="text" initialize-with="."/>`
gives you A. Smith, K. Marx, and A. Beeken which I think is correct for editors in the style, no?